### PR TITLE
Snackbar

### PR DESCRIPTION
**Pull request by khushijohri2001**
Please review my code for my Component Library's Snackbar Component.

The page includes:

Header with Girlifycoder Logo, Heading, and Github link
The navigation bar on the left
Sidebar on the right
Details about the Component and snippet

https://user-images.githubusercontent.com/65448358/160251382-cf5b9b8a-861b-4d98-888f-b2fa696661f0.mp4

### CODE REVIEW

File: snackbar/styles2-snackbar.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  margin: 3rem;
+  height: 30rem;
+}
+
+.snackbar-main-heading {
+  color: yellow;
+}
+.snackbar-sub {
+  margin: 1rem 0rem;
+  color: white;
```
Comment (i-am-a-codegirl) [inline snackbar/styles2-snackbar.css]: use var colors as much as possible

File: snackbar/styles2-snackbar.css [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  font-size: 1.2rem;
+}
+.snackbar-snippet iframe {
+  margin: 1rem 0rem;
+  width: 564px;
+  height: 200px;
```
Comment (i-am-a-codegirl) [inline snackbar/styles2-snackbar.css:18]: do not use px above 10px

File: snackbar/styles2-snackbar.css [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  color: yellow;
+}
+.snackbar-sub {
+  margin: 1rem 0rem;
+  color: white;
+  font-size: 1.2rem;
+}
+.snackbar-snippet iframe {
+  margin: 1rem 0rem;
+  width: 564px;
```
Comment (i-am-a-codegirl) [inline snackbar/styles2-snackbar.css:17]: same use rem or em

**Review by i-am-a-codegirl — APPROVED** [review]

File: snackbar/styles2-snackbar.css [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  height: 200px;
+  border: 0;
+  transform: scale(1);
+  overflow: hidden;
+}
+.standard-snackbar {
+  color: yellow;
```
Comment (pnchinmay) [inline snackbar/styles2-snackbar.css]: These tend to appear different in different browsers
Use color variables with hex code

File: snackbar/styles2-snackbar.css [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+  margin: 1rem 0rem;
+}
+
+.snackbar {
+  background-color: rgb(66, 66, 66);
```
Comment (pnchinmay) [inline snackbar/styles2-snackbar.css:29]: Use only one format for colors, either hex or rgb

**Review by pnchinmay — COMMENTED** [review]
Left some comments
Everything else looks good :100:

**Review by pnchinmay — APPROVED** [review]

File: snackbar/styles2-snackbar.css (hunk 2 above)
Comment (khushijohri2001) [reply snackbar/styles2-snackbar.css:18]: will keep in mind from next time

File: snackbar/styles2-snackbar.css (hunk 1 above)
Comment (khushijohri2001) [reply snackbar/styles2-snackbar.css]: done

File: snackbar/styles2-snackbar.css (hunk 4 above)
Comment (khushijohri2001) [reply snackbar/styles2-snackbar.css]: okayy done

File: snackbar/styles2-snackbar.css (hunk 5 above)
Comment (khushijohri2001) [reply snackbar/styles2-snackbar.css:29]: okay

**Pull request merged**